src/components/MessageList/MessageList.tsx

```diff
@@ -855,12 +855,9 @@ const MessageListWithContext = <
       setFlatListRef(ref);
     }
   };
-
-  const renderEmptyStateIndicator = () => (
-    <View style={styles.flex}>
-      <View style={styles.flex} testID='empty-state'>
-        <EmptyStateIndicator listType='message' />
-      </View>
+  const renderListEmptyComponent = () => (
```

Comment:
This is null for ours but appears inverted if anything is added should we scaleY -1 and make it a component?

Reply:
Also the empty list issue on slack.

Reply:
> This is null for ours but appears inverted if anything is added should we scaleY -1 and make it a component?

hmm I was thinking we should leave this scaleY thingy upto customer side. Since the issue comes from react-native itself. If we add scaleY now, and in next release RN fixes it ... then on previous version on RN its inverted and in next one its correct.

Reply:
Fair enough, any reason it can't be a component instead of a render function then? Also with that logic do we need it to have that view around it at all??